### ISSUE TITLE
[igxPivot] The row chip drop area takes up the whole vertical space, hence you can drop a chip above/below the actual chip area but there's no visual indication where it will go.

### ISSUE THREAD

**Issue by MayaKirova** [IgniteUI/igniteui-angular#10641]
## Description  
The drop area is bigger than the chip area, hence you can drop a chip in it now.
However there's no visual indication where the chip will go.

When over the actual chip area:
![image](https://user-images.githubusercontent.com/10397980/143562794-028f0db1-11b8-49a5-a933-daf8a8eb191f.png)

When somewhere above or below it:
![image](https://user-images.githubusercontent.com/10397980/143563331-ff5a4cff-3a6f-404c-ba4b-3cf4204fbec4.png)

 * igniteui-angular version: 
 * browser: 

## Steps to reproduce  

1. Move a column chip over the row area but outside of the actual chip area:
![image](https://user-images.githubusercontent.com/10397980/143563016-09899b30-5423-4ab5-8c0c-8f0fad62a1c9.png)
2. Notice that there's no indication that it will be added anywhere.
3. Drop it and it will get added to the end of the chip area.

## Result  
The chip is added but there was no visual indication that dropping it will add it.

## Expected result  
Either the chip should not be added or there should be a visual indication that it will be added at the end when you are over the row chip area.

## Attachments  
Attach a sample if available, and screenshots, if applicable.

**Comment by MayaKirova** [comment]
@dkamburov 
I guess this should be discussed with @desig9stein and @StefanIvanov. Was the drop container expanded to fill the vertical space unintentionally as part of the styling or do we actually want to be able to drop a chip there? If we want to drop a chip there what should we show as indication?

**Comment by StefanIvanov** [comment]
We can take advantage of the enhancements we discussed yesterday that are in the updated spec 3.1 last image

**Comment by MayaKirova** [comment]
@StefanIvanov 
I've taken a look but I'm not sure that is the best option since you can currently put the chip between any of the existing chips (not just at the end) and there's indicators that shows exactly where it will go ( the red line). It is similar to the design in the wpf pivot and it is different than, for example the group by chip area where you cannot put a new column between existing chips. For group by you need to drop it at the end first and then as a second step reorder it.

If you'd prefer we can scrap the existing functionality that allows dropping directly between existing chips and do it like in the group by chip area, but to me the wpf behavior seems better since you don't need to do 2 steps to order the chips the way you want.

Also in the last image it looks like you can move something that represents a dimension to the values area. In general values cannot be used as dimensions and vice versa (as one represents groupable fields and the other aggregates) and it is a limitation that exists in all of our pivot grids (igPivot, wpf pivot etc.). Not sure if this is something you wish to change for the igxPivotGrid or its just an oversight.

**Comment by StefanIvanov** [comment]
I think we should strive for consistent behavior between grids when it comes to dragging and dropping chips in attached areas. Is there a particular reason/limitation for the group by that enforced the current behavior?

**Comment by MayaKirova** [comment]
I don't think there's a technical limitation for the group by area. It can be implemented то behave the same as the pivot chips.
It was probably a UX decision made way before we've considered the existing UX in the wpf pivot grid.

Other differences are that the group area reorder is immediate (while you are still dragging the chip the other chips reorder and the grid reflects the change) vs in the pivot where it is deferred  (you can move the chip around and it shows an indicator where it will go but the change is applied only after you drop it).

**Comment by MayaKirova** [comment]
Closing since the new design will allow dropping anywhere in the area and indication will be an additional chip with "Drop here" text.
